src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -218,9 +217,9 @@ export default function TokenDetails({
 
         <RightPanel>
           <Widget
-            token={token ?? undefined}
-            defaultField={Field.OUTPUT}
-            onTokenChange={navigateToWidgetSelectedToken}
+            outputToken={token ?? undefined}
```

Comment:
Why do you have `outputToken` _and_ `defaultToken` here? Will the token details "page token" now always match the output token?

Reply:
no, the page token can be switched to input. we only navigate (to output) when the page token is replaced.

i set both props because I thought it was clearer, instead of assuming that the default token will be assigned to the Field you expect. 

e.g. this API would support a situation where the Input is the "default" (i.e. required) token, if you specify all 3 props like this:

```
inputToken={USDC}
outputToken={DAI}
defaultToken={USDC}
```

but since we do automatically assign default -> output, we can remove one here like you suggest